packages/scandipwa/src/route/ProductPage/ProductPage.component.js

```diff
@@ -132,31 +164,90 @@ export class ProductPage extends PureComponent {
         );
     }
 
-    renderAdditionalSections() {
+    renderProductInformationTab(key) {
         const {
             dataSource,
             parameters,
             areDetailsLoaded
         } = this.props;
 
+        return (
+            <ProductInformation
+              product={ { ...dataSource, parameters } }
+              areDetailsLoaded={ areDetailsLoaded }
+              key={ key }
+            />
+        );
+    }
+
+    renderProductAttributesTab(key) {
+        const {
+            dataSource,
+            parameters,
+            areDetailsLoaded
+        } = this.props;
+
+        return (
+            <ProductAttributes
+              product={ { ...dataSource, parameters } }
+              areDetailsLoaded={ areDetailsLoaded }
+              key={ key }
+            />
+        );
+    }
+
+    renderProductReviewsTab(key) {
+        const {
+            dataSource,
+            areDetailsLoaded
+        } = this.props;
+
+        return (
+            <ProductReviews
+              product={ dataSource }
+              areDetailsLoaded={ areDetailsLoaded }
+              key={ key }
+            />
+        );
+    }
+
+    renderProductTabItems() {
+        return Object.values(this.tabMap).reduce((tabRenders, { shouldTabRender, render, name }) => {
+            if (!shouldTabRender()) {
+                tabRenders.push(render(name));
```

Comment:
You could use a function here returning a Fragment with a key, instead of passing key into each function

Reply:
Please note this for the future.